Firmware/ultralcd.cpp

```diff
@@ -3423,10 +3411,7 @@ static void lcd_silent_mode_set() {
   // Wait until the planner queue is drained and the stepper routine achieves
   // an idle state.
   st_synchronize();
-  if (tmc2130_wait_standstill_xy(1000)) {}
```

Comment:
Proposal for change:

Guard the `lcd_display_message_fullscreen_P` call by `blocks_queued()`. If there are no planned moves, there is no need to render this message, it creates a weird LCD draw noticable by the user.

This does add about 10B of flash but I think its worth it for slightly cleaner UI. I'm not sure if `tmc2130_wait_standstill_xy` should be guarded as well, you're more familiar with it than me.

![image](https://user-images.githubusercontent.com/8218499/233787897-d0ae5e0e-7523-49e7-b8ae-8e6bdcb67a79.png)